### PR TITLE
dotfiles

### PR DESCRIPTION
**Pull request by kzidane**
Per https://github.com/cs50/ide50/issues/77!

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,7 +38,7 @@ deb: clean Makefile
 
 	# set permissions
 	chmod -R 755 "$(FILES_DIR)/usr/bin/"
-	chmod 644 "$(VERSION_FILE)" "$(FILES_DIR)/etc/profile.d/ide50.sh" "$(FILES_DIR)/home/ubuntu/.prompt50"
+	chmod 644 "$(VERSION_FILE)" "$(FILES_DIR)/etc/profile.d/ide50.sh" "$(FILES_DIR)/home/ubuntu/.cs50/.prompt50"
```
Comment (dmalan) [inline Makefile]: Can probably be just `prompt` now instead of `.prompt50`?

File: files/usr/bin/hostname50 [hunk 2]
```diff
@@ -7,7 +7,7 @@ if [ "$IDE_OFFLINE" ]; then
 fi
 
 # determine external host that IDE is running on to calculate app domain
-source /home/ubuntu/.domain50 &>/dev/null
+source /home/ubuntu/.cs50/stats/.domain50 &>/dev/null
```
Comment (dmalan) [inline files/usr/bin/hostname50]: Just `domain` here?

File: files/usr/bin/password50 [hunk 3]
```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
-password_file=/home/ubuntu/.password50
+password_file=/home/ubuntu/.cs50/.password50
+if [ -f /home/ubuntu/.password50 ]; then
+    mv /home/ubuntu/.password50 $password_file &>/dev/null
```
Comment (dmalan) [inline files/usr/bin/password50]: Should this not be `/home/ubuntu/.cs50/password` now?

File: postinst [hunk 4]
```diff
@@ -104,9 +104,12 @@ if ! grep -qs "alias cd" /home/ubuntu/.bashrc; then
 fi
 
 if ! grep -qs 'prompt50' /home/ubuntu/.bashrc; then
-    echo "source /home/ubuntu/.prompt50" >>/home/ubuntu/.bashrc
+    echo "source /home/ubuntu/.cs50/.prompt50" >>/home/ubuntu/.bashrc
```
Comment (dmalan) [inline postinst]: `/home/ubunbut/.cs50/prompt`?

File: postinst [hunk 5]
```diff
@@ -104,9 +104,12 @@ if ! grep -qs "alias cd" /home/ubuntu/.bashrc; then
...
 fi
 
+# update path in existing .bashrc files
+sed -i "s#\(/home/ubuntu/\)\(.prompt50\)#\1.cs50/\2#" /home/ubuntu/.bashrc
```
Comment (dmalan) [inline postinst]: Renamings here too?

**Comment by kzidane** [comment]
Thanks for approving! I'll merge once I'm done testing since some of these files are generated by other plugins.

**Comment by kzidane** [comment]
Base on https://github.com/c9/harvard.cs50/pull/84!

File: update50 [hunk 6]
```diff
@@ -11,8 +11,9 @@ fi
 # suppress CS50 compilation preferences
 unset CC CFLAGS LDLIBS
 
-# remove old .info50 and update50 if they exists
+# remove old files if new update50 exists
 if [ -f /home/ubuntu/.cs50/bin/update50 ]; then
+    rm -f /home/ubuntu/{.client50,.domain50,.password50,.prompt50}
```
Comment (dmalan) [inline update50]: Should we redirect `stderr` to `/dev/null` in case one of these doesn't exist?

Comment (kzidane) [inline update50]: `-f` causes `rm` to ignore nonexistent files, so it doesn't throw any errors.

Comment (dmalan) [inline update50]: 👍

**Pull request merged**